src/components/ApprovalWorkflowSection.tsx

```diff
@@ -17,19 +15,16 @@ type ApprovalWorkflowSectionProps = {
     /** Single workflow displayed in this component */
     approvalWorkflow: ApprovalWorkflow;
 
-    /** ID of the policy */
-    policyID: string;
+    /** A function that is called when the section is pressed */
+    onPress: () => void;
 };
 
-function ApprovalWorkflowSection({approvalWorkflow, policyID}: ApprovalWorkflowSectionProps) {
+function ApprovalWorkflowSection({approvalWorkflow, onPress}: ApprovalWorkflowSectionProps) {
```

Comment:
@blazejkustra Why do you make this change? I think `onPress` function could be handled in ApprovalWorkflowSection.

Reply:
I prefer this section to be controlled by the parent, so no business logic is built into it